bellows/ezsp.py

```diff
@@ -143,7 +143,7 @@ def cb(frame_name, response):
 
     def __getattr__(self, name):
         if name not in self.COMMANDS:
-            raise AttributeError
+            raise AttributeError("%s not found in COMMANDS" % name)
```

Comment:
`name` is unknown input here. It could be a tuple, which could lead to:

```
>>> "This could blow up %s" % ("unexpectedly", "hah")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
```

It'd be safer to write this as

  raise AttributeError("%s not found in COMMANDS" % (name, ))